apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -220,14 +324,31 @@ export function buildMachine(
       events: {} as PaperHandlerStatusEvent,
     },
     id: 'bmd',
-    initial: 'no_paper',
-    context: {
-      workspace,
-      driver,
-      pollingIntervalMs,
-    },
+    initial: 'not_accepting_paper',
+    context: initialContext,
     states: {
-      no_paper: {
+      // Initial state. Doesn't accept paper and transitions away when the frontend says it's ready to accept
+      not_accepting_paper: {
```

Comment:
It doesn't automatically grab paper but ejecting paper in this state is useful if the device or app crashes during or after the print phase. I think this state is more useful for development but I'm inclined to keep it in since it doesn't add any new actions, but open to other thoughts.

Reply:
Ah yeah that makes sense to have it startup gracefully no matter what physical state the machine is in. A comment to that effect might be helpful for posterity

Reply:
[0f2aee4](https://github.com/votingworks/vxsuite/pull/3855/commits/0f2aee42cc9d09d86f77815eabeb46b545d9f3c7)